setup.examples/exim/etc/MailScanner/conf.d/mailwatch.conf

```diff
@@ -0,0 +1,29 @@
+# Example config for exim  /etc/MailScanner/conf.d/mailwatch.conf
```

Comment:
In fact, I'm wrong to try to put in /etc/MailScanner/conf.d/mailwatch.conf "personal" informations.

We just need to have in mailwatch.conf the essential values needed by MailWatch and say to the user to define another file as  /etc/MailScanner/conf.d/server.conf where he will put is own personal informations.

If we do this, may be this is not needed in mailwatch.conf:

```
Max Unscanned Messages Per Scan = 50
 Max Unsafe Messages Per Scan = 50
 Max Normal Queue Size = 2000
 Deliver Unparsable TNEF = yes
 Find UU-Encoded Files = yes
 Max Children = 10
```

To check.